airqo-platform/AirQo-frontend
# Add static code analysis and linting

#### Summary of Changes (What does this PR do?)
- Adds static code analysis and linting

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Pull changes in this PR to your local computer
* Make a change in the platform folder
* You should have the files automatically linted for you

#### What are the relevant tickets?
- [PLAT-1228](https://airqoteam.atlassian.net/browse/PLAT-1228?atlOrigin=eyJpIjoiMzQxYThkMWE4NWJmNGJhYzgyMGJkZjJkZjBhOGNkMjciLCJwIjoiaiJ9)

#### Screenshots (optional)
